### PR TITLE
Improve respec autolinking

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -57,9 +37,9 @@
 <body>
   <section id="abstract">
     <p>
-      This specification extends <a>MediaStreamTrack</a> to provide a
+      This specification extends {{MediaStreamTrack}} to provide a
       media-content hint attribute. This optional hint permits
-      <a>MediaStreamTrack</a> consumers such as <dfn>PeerConnection</dfn>
+      {{MediaStreamTrack}} consumers such <dfn>RTCPeerConnection</dfn>
```
Comment (alvestrand) [inline index.html]: an "as" got eaten here.

File: index.html [hunk 2]
```diff
@@ -212,27 +192,27 @@ <h2>Extension to MediaStreamTrack</h2>
...
           best-informed guess on how to handle contained audio data. This may be
           inferred from how the track was opened or by doing content analysis.
-        <tr><td><code id="idl-def-AudioContentHint.speech">speech</code></td><td><p>
+        <tr><td><dfn id="idl-def-AudioContentHint.speech"><code>"speech"</code></dfn></td><td><p>
           The track should be treated as if it contains speech data. Consuming
           this signal it may be appropriate to apply noise suppression or boost
           intelligibility of the incoming signal.
         </p></td></tr>
-        <tr><td><code id="idl-def-AudioContentHint.speechRecognition">speechRecognition</code></td><td><p>
+        <tr><td><dfn id="idl-def-AudioContentHint.speechRecognition"><code>"speechRecognition"</code></dfn></td><td><p>
```
Comment (alvestrand) [inline index.html:208]: This one is changing to "speech-recognition" per other PR.

**Review by alvestrand — APPROVED** [review]
I saw only one place where a word was missing. I'll trust you on the headers.

File: index.html (hunk 1 above)
Comment (dontcallmedom) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (dontcallmedom) [reply index.html:208]: right - I'm happy to integrate that PR in this one, or we can manage the conflict once one of them gets merged

**Pull request merged**